updater/internal/updater/updater.go

```diff
@@ -16,10 +16,12 @@
 package updater
```

Comment:
I did some testing. 

1. Upgrading a system using the install script - This worked. 
- On a VM, I installed an agent in Stage and verified basic functionality. 
- I copied the `.deb` package from [here](https://github.com/observIQ/bindplane-otel-collector/actions/runs/15903539822) onto the VM
- I ran the install script with the --file flag pointed at the new `.deb` package
- The agent updated fine and its service file showed the group updated to `bdot`
- The user was unchanged since it installed as root.